src/garage/envs/bullet/bullet_env.py

```diff
@@ -1,19 +1,36 @@
 """Wrappers for py_bullet environments."""
 import inspect
+import math
 
 import akro
 import gym
-from pybullet_envs.bullet.minitaur_duck_gym_env import MinitaurBulletDuckEnv
-from pybullet_envs.bullet.minitaur_gym_env import MinitaurBulletEnv
-from pybullet_envs.env_bases import MJCFBaseBulletEnv
+from gym.wrappers.time_limit import TimeLimit
 
+from garage import Environment, StepType, TimeStep
 from garage.envs.env_spec import EnvSpec
 
-
-class BulletEnv(gym.Wrapper):
+# The bullet gym environments in this list inherit from the
+# MJCFBasedBulletEnv class, which doesn't follow a consistent class variable
+# naming practice -> constructor param `render` is stored as `isRender`. Thus
+# they require additional pickling logic.
+MJCFBASED_BULLET_ENV = [
+    'ReacherBulletEnv', 'PusherBulletEnv', 'StrikerBulletEnv',
+    'ThrowerBulletEnv', 'Walker2DBulletEnv', 'InvertedPendulumBulletEnv',
+    'InvertedDoublePendulumBulletEnv', 'InvertedPendulumSwingupBulletEnv',
+    'HalfCheetahBulletEnv', 'AntBulletEnv', 'HopperBulletEnv',
+    'HumanoidBulletEnv', 'HumanoidFlagrunBulletEnv',
+    'HumanoidFlagrunHarderBulletEnv'
+]
+
+
+class BulletEnv(Environment):
     """Binding for py_bullet environments."""
 
-    def __init__(self, env=None, env_name='', is_image=False):
+    def __init__(self,
+                 env=None,
```

Comment:
perhaps accept a positional argument `env` and then dispatch on whether it's a string or a `gym.Env` (this makes the interface a little more ergonomic)